wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -675,7 +678,9 @@ public void deleteProject(
   @ApiOperation(value = "Get workspace server configuration values")
   @ApiResponses({@ApiResponse(code = 200, message = "The response contains server settings")})
   public Map<String, String> getSettings() {
-    return new HashMap<>();
+    return ImmutableMap.of(
+        Constants.SUPPORTED_ENVIRONMENTS,
```

Comment:
I'd change constant according to value which will be there -> `supportedRecipes`